doc/How-it-opens-files.xml

```diff
@@ -11,11 +11,20 @@
   <refnamediv> <refname>Opening</refname> <refpurpose>How libvips opens files</refpurpose> </refnamediv>
 </para>
 <para>
-  libvips now has at least four different ways of opening image files, each best for different file types, file sizes and image use cases. libvips tries hard to pick the best strategy in each case and mostly you don’t need to know what it is doing behind the scenes, except unfortunately when you do.
```

Comment:
These files are being used when pandoc is not available (if building with `-Dgtk_doc=true`):
https://github.com/libvips/libvips/blob/8ae3e5eb9a8075b4951cb2d928b5a455bfe96fc8/doc/meson.build#L95-L105

We could make pandoc a required dependency, but I'm aware that some distros doesn't distribute pandoc as it depends on [many Haskell dependencies](https://src.fedoraproject.org/rpms/pandoc/blob/fb1cbb02adaa7a2e05fa7159fac6c0e2bde20409/f/pandoc.spec#_60-288).

It's a bit unfortunate that we still need this Markdown to DocBook conversion. It would be much easier if gtk-doc supported Markdown out-of-the-box.

Reply:
Ooof, I'd forgotten, sigh.